cdc-accelerator/feature/permissionGroups.js

```diff
@@ -45,16 +45,63 @@ export default class PermissionGroups extends PartnerFeature {
         this.copyFileFromSrcToBuild(srcFeaturePath, PermissionGroups.PERMISSIONGROUP_FILE_NAME)
     }
 
-    async deploy(directory) {
-        console.log('deploy called')
+    async deploy(partnerDirectory, siteInfo) {
```

Comment:
I have created a new group on src/permissiongroups.js and saw in the output that the group already exists!
Group bruno_test already exists. Skipping...